.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
```

Comment:
Not required. We have strict list of dependencies which are well known and covered by tests. Any new update initiated by dependabot requires code/tests change and etc.

Reply:
I use dependabot to notify me of updates and move forward if tests don't fail as not every update requires changes. Breaking changes should be catched by the pipeline. To me this is a good way to automatically move forward with new versions and reduce technical debt. I thought adding it would be consistent as it is enabled on the frontend repo.

Reply:
Frontend lifecycle not strict to backend lifecycle. This how the technologies evolves. We cannot invest time of our backend engineers to fix non stable versions each and every time after minor release of some small pypi dependency.